lib/rubocop/cop/minitest/refute_equal.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Check if your test uses `assert` with negation instead
+      # of `refute_equal(expected, actual)`.
+      #
+      # @example
+      #   # bad
+      #   assert(!"rubocop-minitest".eql?(actual))
```

Comment:
FYI, I opened a PR https://github.com/rubocop-hq/minitest-style-guide/pull/20.
`refute_equal` method uses `==` method, not `eql?` method.
https://github.com/seattlerb/minitest/blob/15ed8e4/lib/minitest/assertions.rb#L587-L592

`==` method and `eql?` method behave differently.

Reply:
interesting.. so, what do you think to keep thi Cop to only check  for `assert(expected != actual)` ? 🤔

Reply:
Probably `assert(expected != actual)` and `assert(!expected == actual)` will be checked. Let's first look at PR rubocop-hq/minitest-style-guide#20 trends.

Reply:
FYI, https://github.com/rubocop-hq/minitest-style-guide/pull/20 has been merged.